File: src/main/java/org/tdl/vireo/model/repo/GraduationMonthRepo.java
package org.tdl.vireo.model.repo;

import org.tdl.vireo.aspect.annotation.EntityCV;
import org.tdl.vireo.model.GraduationMonth;
import org.tdl.vireo.model.repo.custom.GraduationMonthRepoCustom;

import edu.tamu.weaver.data.model.repo.WeaverOrderedRepo;

@EntityCV(name = "Graduation Months")
public interface GraduationMonthRepo extends WeaverOrderedRepo<GraduationMonth>, EntityControlledVocabularyRepo<GraduationMonth>, GraduationMonthRepoCustom {

    public GraduationMonth findByMonth(int month);

}

File: src/main/webapp/WEB-INF/app/repo/graduationMonthRepo.js
vireo.repo("GraduationMonthRepo", function (api, GraduationMonth, WsApi) {

	var GraduationMonthRepo = function() {
		var graduationMonthRepo = this;
		graduationMonthRepo.model = GraduationMonth;
		graduationMonthRepo.mapping = api.GraduationMonth;
		return graduationMonthRepo;
	};

	return new GraduationMonthRepo();
	
});